services/ansibler/server/ansible-playbooks/wireguard/tasks/configure.yml

```diff
@@ -48,6 +48,12 @@
 - name: Set MTU size for wg0
   ansible.builtin.shell: ifconfig {{ wg_interface_name }} mtu {{ mtu_size }}
 
+- name: Ensure file /etc/dhcp/dhclient.conf exists
+  ansible.builtin.copy:
+    content: ""
+    dest: /etc/dhcp/dhclient.conf
+    force: false
+
```

Comment:
Why do we need this ?

Reply:
Yes, it is missing in Ubuntu 24.04 and the playbook failed due to that.